custom_components/enphase_envoy/envoy_endpoints.py

```diff
@@ -2,7 +2,7 @@
     # Generic endpoints
     "info": {
         "url": "https://{}/info.xml",
-        "cache": 86400,
+        "cache": 3600,
```

Comment:
Was this a deliberate change, Vincent?

Reply:
Yes it is :) The envoy firmware version is read from this file. Since there was no sensor using this data before we would just update once a day. Since we now have the firmware sensor I think we should update it at least once an hour for the sensor to be useful.

Reply:
Good point.